storio-common-annotations-processor/src/test/java/com/pushtorefresh/storio/common/annotations/processor/introspection/JavaTypeTest.java

```diff
@@ -39,6 +41,29 @@ private static TypeMirror mockTypeMirror(@Nullable TypeKind typeKind, @Nullable
 
         return typeMirror;
     }
+    
+    @Test(expected = IllegalArgumentException.class)
```

Comment:
Can you use please ExpectedException or try-catch-fail to check cause and exception message?